<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Пользователи онлайн</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Сообщения</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionNew_User"/>
    <addaction name="actionControl_users"/>
    <addaction name="actionControl_docs"/>
   </widget>
   <widget class="QMenu" name="menuClient">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Client</string>
    </property>
    <addaction name="actionMake_New_Doc"/>
    <addaction name="actionNew_document"/>
    <addaction name="actionDownload_documents"/>
   </widget>
   <addaction name="menuServer"/>
   <addaction name="menuClient"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClient"/>
   <addaction name="actionServer"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Client: OFF</string>
   </property>
   <property name="toolTip">
    <string>Launch client</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Server: OFF</string>
   </property>
   <property name="toolTip">
    <string>Launch server</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_User">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New User</string>
   </property>
  </action>
  <action name="actionControl_users">
   <property name="text">
    <string>Control users</string>
   </property>
  </action>
  <action name="actionControl_docs">
   <property name="text">
    <string>Control docs</string>
   </property>
  </action>
  <action name="actionNew_document">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>My document control</string>
   </property>
  </action>
  <action name="actionMake_New_Doc">
   <property name="text">
    <string>Make New Doc</string>
   </property>
  </action>
  <action name="actionDownload_documents">
   <property name="text">
    <string>Download documents</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
